app/helpers/application_helper.rb

```diff
@@ -307,6 +307,15 @@ def name_title(args)
     end
   end
 
+  def link_to_search_value(args)
+    if args[:document][args[:field]].present?
+      args[:document][args[:field]].each_with_index do |field_t, i|
+        field_link = link_to(field_t, "/?f[#{args[:field]}][]=#{CGI.escape field_t}", class: 'search-name', 'data-toggle' => 'tooltip', 'data-original-title' => "Search: #{field_t}", title: "Search: #{field_t}")
+        args[:document][args[:field]][i] = field_link.html_safe if field_link
```

Comment:
the `if field_link` check isn't necessary. Neither is the `html_safe` because the `link_to` method returns an html-safe string (and allows the link html to render correctly)